website/src/pages/en/subgraphs/cookbook/subgraph-composition-three-sources.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Aggregate Data Using Subgraph Composition
+sidebarTitle: 'Build a Composable Subgraph with Multiple Subgraphs'
+---
+
+Optimize your Subgraph by merging data from three independent, source Subgraphs into a single composable Subgraph to enhance data aggregation.
+
+> Important Reminders:
+>
+> - Subgraph composition is built into the CLI, and you can deploy with [Subgraph Studio](https://thegraph.com/studio/).
+> - This feature requires `specVersion` 1.3.0.
+
+## Overview
+
+Subgraph composition empowers you use one Subgraph as a data source for another, allowing it to consume and respond to entity changes. Instead of fetching on-chain data directly, a Subgraph can listen for updates from another Subgraph and react to changes. This is useful for aggregating data from multiple Subgraphs or triggering actions based on external updates.
+
+## Prerequisites
+
+To deploy **all** Subgraphs locally, you must have the following:
+
+- A [graph-node](https://github.com/graphprotocol/graph-node) instance running locally
```

Comment:
Nitpick, but I feel like if we write it "graph-node" it should be inline code, but if we write it "Graph Node" it should be regular text. What do you think? So I'm proposing either:
```suggestion
- A [`graph-node`](https://github.com/graphprotocol/graph-node) instance running locally
```
or
```suggestion
- A [Graph Node](https://github.com/graphprotocol/graph-node) instance running locally
```

Reply:
(Note that we use "Graph Node" below, on line 34.)